src/main/java/ua/com/conductor/model/dto/MovieSessionUpdateDto.java

```diff
@@ -0,0 +1,40 @@
+package ua.com.conductor.model.dto;
+
+public class MovieSessionUpdateDto {
```

Comment:
Are you sure you need this DTO? I do not think that you will receive different DTOs from the request